terraform/aws/output.tf

```diff
@@ -3,12 +3,12 @@ output "public_ip" {
   description = "The public IPs address of the provisioned node(s)"
 }
 
-output "private_dns" {
+output "instance_private_dns" {
   value       = module.instances.private_dns
   description = "The private DNS of the provisioned node(s)"
 }
 
-output "private_ips" {
+output "private_ip" {
```

Comment:
we should keep it plural as user can spin up more then 1 instance which is depends on node_count variable.